app/assets/stylesheets/application.scss

```diff
@@ -25,6 +25,7 @@
 @import "honeycrisp-compact";
 @import "components/actions-wrapper";
 @import "components/checkmark-list";
+@import "components/icons";
```

Comment:
I originally put this file under atoms/ but Jenny reminded me that we've been moving away from that organizing system in favor of putting everything in either components/ or templates/.